python/pyspark/pandas/frame.py

```diff
@@ -6318,10 +6318,7 @@ def pivot_table(
                 )
                 psdf = DataFrame(internal)
         else:
-            if isinstance(values, list):
-                index_values = values[-1]
```

Comment:
This condition checks if the `index is None` and `isinstance(values, list)`.

However, it's already filtered in the above path:
```python
        if isinstance(values, list) and index is None:
            raise NotImplementedError("values can't be a list without index.")
```

So this condition never be the `True`, until we implement the missing support.